portality/static/css/doaj.css

```diff
@@ -77,6 +77,14 @@ blockquote {
     background-color: white;
 }
 
+.navbar .nav .active > a:focus,             
+.navbar .nav > li:focus,
+.navbar .nav > li > a:focus,
+.navbar .navbar-header a:focus {
+    color: #F68B1F;
+    background-color: #eeeeee;
```

Comment:
It probably should be `background-color: white`. This way you will have the same background for :focus and for :hover

Reply:
![Screenshot from 2019-10-23 10-22-18](https://user-images.githubusercontent.com/8298769/67378935-f1545b80-f57f-11e9-9ede-e43311448f39.png)

Reply:
Yes, so, interesting point.  That was my first consideration, but an active nav is also highlighted white, so is there a usability question around having the tab-based navigation highlight in the same way as the current active nav?  That is, if you tab through the nav, it's not possible to visually determine which is the current nav and which is the focused nav item?

Having said that, the grey looks rubbish, and I notice that when you click a link it briefly becomes grey before becoming white.

Perhaps an additional signifier, like also underlining the focused one, as well as highlighting in white?

Reply:
I agree, there should be different colours for focus and hover - you want navigating to the next item to be different to what you're currently on. The font colour change makes it readable and that's a good start.

Perhaps we want a lighter orange?